### ISSUE TITLE
Browser will not render svg element when wrapped by Radium

### ISSUE THREAD

**Issue by MeanwhileMedia** [FormidableLabs/radium#189]
For some reason, when I use Radium on the below React class, the browser does not render it (prob because it spits out invalid markup) then React gives me "Uncaught Error: Invariant Violation" because the dom and virtual dom don't match.

```
define([
  'React',
  'Radium',
  'underscore'
], function (React, Radium, _){

  var BbAppTreeNode = React.createClass({
    render: function(){
      return(
        <g className='node'>
          <circle r='4.5'></circle>
          <text x="-10" dy=".35em" textAnchor="end">{this.props.data.name}</text>
        </g>
      )
    }
  })

  return Radium.Enhancer(BbAppTreeNode)

})
```

I can get the class to render if:
1. I remove {this.props.data.name} from the text element. Leaving it empty.
2. I don't wrap it in Radium.Enhancer

I'm using Radium v0.12.0 and React v0.13.3

**Comment by ianobermiller** [comment]
What type are you passing as this.props.data.name? I cannot reproduce this: http://jsbin.com/vijawaveqa/1/edit?html,js,output

**Comment by MeanwhileMedia** [comment]
Hey @ianobermiller thanks for checking it out. this.props.data.name is a string. I saw your jsbin, and I'm still not sure what differs with my implementation. 

However, It looks like somehow the data.name text is getting wrapped in a span (on my side, and in your jsbin). When/why this span is getting added? I don't know... but it only gets added when I use radium.  I don't think that is allowed, right? https://developer.mozilla.org/en-US/docs/Web/SVG/Element/text

**Comment by ianobermiller** [comment]
Thanks for bringing that up! The span thing is a problem everywhere, not just SVG (it only happens to break SVG, but it is adding extra DOM nodes for everywhere else).

**Comment by MeanwhileMedia** [comment]
Thanks for several great fixes in v0.13.0 @ianobermiller

**Comment by ianobermiller** [comment]
My pleasure :)
